hw/sensor/creator/src/sensor_creator.c

```diff
@@ -508,6 +543,51 @@ config_bma253_sensor(void)
 }
 #endif
 
+#if MYNEWT_VAL(BMA2XX_OFB)
+/**
+ * BMA2XX sensor default configuration
+ *
+ * @return 0 on success, non-zero on failure
+ */
+int
+config_bma2xx_sensor(void)
+{
+    struct os_dev * dev;
+    struct bma2xx_cfg cfg;
+    int rc;
+
+    dev = os_dev_open("bma2xx_0", OS_TIMEOUT_NEVER, NULL);
+    assert(dev != NULL);
+
+    cfg.model = BMA2XX_BMA280;
```

Comment:
How I should use it if I have BMA253 or event both BMA280 and BMA253?

Reply:
There is two ways for this first implementation:
1.  Since Vipul's original BMA253 will persist alongside BMA2XX for a while the user can define both the BMA2XX_OFB and BMA253_OFB flags and two separate devices will be created.
2.  If the user wanted to just use the BMA2XX and create both a BMA280 and BMA253 sensor they would need to add a second device, bma2xx_cfg and sensor_itf in the appropriate BMA2XX_OFB sections.

Makes sense?

Reply:
The issue I see here is that sensor_creator.c creates only one `bm2xx_0`, so if somebody want to make use of this driver for more then one sensor he needs to modify sensor_creator.c.  It would be nice to have possibility to configure it in other way. Anyway, this can be handled later.